src/Components/Patient/SampleTest.tsx

```diff
@@ -66,10 +75,39 @@ export const SampleTest = (props: any) => {
   const { facilityId, patientId } = props;
   const [state, dispatch] = useReducer(sampleTestFormReducer, initialState);
   const [isLoading, setIsLoading] = useState(false);
+  const [facilityName, setFacilityName] = useState<Array<FacilityNameModel>>(
+    []
+  );
 
   const headerText = "Request Sample";
   const buttonText = "Confirm your request to send sample for testing";
 
+  const fetchFacilityName = useCallback(
+    async (status: statusType) => {
+      const facility_type = 950;
+      const FacilityNameList = await dispatchAction(
+        getAllFacilities({ facility_type })
+      );
+      if (!status.aborted && FacilityNameList.data.results) {
+        setFacilityName([...FacilityNameList.data.results]);
+        dispatch({
+          type: "set_form",
+          form: {
+            ...state.form,
+            testing_facility: FacilityNameList.data.results[0]?.id,
```

Comment:
Codacy found an issue: [Identifier 'testing_facility' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5838477)